store/sqlstore/utils.go

```diff
@@ -147,3 +147,18 @@ func constructArrayArgs(ids []string) (string, []interface{}) {
 
 	return "(" + placeholder.String() + ")", values
 }
+
+// morphWriter is a target to pass to the logger instance of morph.
+// For now, everything is just logged at a debug level. If we need to log
+// errors/warnings from the library also, that needs to be seen later.
+type morphWriter struct {
+}
+
+func newMorphWriter() *morphWriter {
+	return &morphWriter{}
+}
```

Comment:
Do we need this function? It seems to be redundant.

Reply:
Hmm okay, will remove.